kdestandardize.ado

```diff
@@ -234,6 +234,34 @@ prog def kdestandardize
 				   9 `"INFORMATION SUPPORT AND SERVICES"'		 			 ///
 				   10 `"TECHNOLOGY"', modify
 				   
+	   // Define PERFORMANCE_TYPE
```

Comment:
For binary indicators you should use 0 and 1.  With a value of 1 and 2, this becomes unusable without further cleaning if entered into a regression model where a value of 0 is assumed to exist for all the covariates.  Additionally, when the value labels are short/few you can put them on a single line:

```Stata
la def assesslev 0 "Nation" 1 "Kentucky", modify
```

to avoid needing as much space.